meta/main.yml

```
@@ -26,6 +26,9 @@ galaxy_info:
       versions:
         - bionic
         - focal
+    - name: Debian
+      versions:
+        - 11
```

`ansible-lint` expects a string here.

```suggestion
        - "11"
```